code/modules/mob/living/silicon/ai/ai.dm

```diff
@@ -338,16 +338,26 @@ var/list/ai_list = list()
 /mob/living/silicon/ai/proc/ai_roster()
 	show_station_manifest()
 
-/mob/living/silicon/ai/proc/ai_call_shuttle()
+/mob/living/silicon/ai/proc/ai_call_or_recall_shuttle()
 	if(isDead())
-		to_chat(src, "You can't call the shuttle because you are dead!")
+		to_chat(src, "You can't call/recall the shuttle because you are dead!")
 		return
 	if(istype(usr,/mob/living/silicon/ai))
 		var/mob/living/silicon/ai/AI = src
 		if(AI.control_disabled)
 			to_chat(usr, "Wireless control is disabled!")
 			return
+	switch(emergency_shuttle.direction)
+		if(EMERGENCY_SHUTTLE_RECALLED)
+			to_chat(usr, "Wait until the shuttle arrives to Centcomm and try again")
```

Comment:
all 3 messages in this class need spans, also grammar here could use work
`at` not `to`, plus add a period.